app/views/layouts/_navbar.html.erb

```diff
@@ -45,12 +49,18 @@
         <li class="nav-item <%= request.fullpath.include?('stock') ? 'active' : '' %>">
           <%= link_to "Stocks", stocks_path, class: "nav-link" %>
         </li>
+        <li class="nav-item">
```

Comment:
Delete duplicate "Edit Account" later on. Add conditional logic for active class.